strapi/strapi
# feat: add publish & update action

### What does it do?

* adds addDocumentAction API
* adds publish & update action
* refactors DescriptionComponentRenderer for better rendering (less popping, more wait till everyone is ready)
* adds published tab

### How to test it?

* You should be able to create & save an entry
* You should be able to publish that entry

### Related issue(s)/PR(s)

* resolves CONTENT-2018
* resolves CONTENT-2019
* resolves CONTENT-2020
* resolves CONTENT-2022
* resolves CONTENT-2023
* resolves CONTENT-2113
